### ISSUE TITLE
Playlist: relative Path not working when going up two levels

### ISSUE THREAD

**Issue by sebmuc99** [music-assistant/support#2291]
### What version of Music Assistant has the issue?

2.0.3

### Have you tried everything in the Troubleshooting FAQ and reviewed the Open and Closed Issues and Discussions to resolve this yourself?

- [X] Yes

### The problem

relative Path in local Playlists not working

![image](https://github.com/music-assistant/hass-music-assistant/assets/32903413/54ecea6f-4a8a-4640-ac5a-a203ed2a4cc7)

### How to reproduce

Use relative Path in Playlists

### Music Providers

local disc

### Player Providers

any

### What version of Home Assistant Core are your running

2024.5.3

### What type of installation are you running?

Home Assistant Core

### On what type of hardware are you running?

Raspberry Pi

**Comment by marcelveldt** [comment]
Relative paths are supported but not in an higher level than the folder you added to the file provider.

**Comment by Rudd-O** [comment]
A bit of context is needed:

The container mounts your music on `/media`.  Naturally, any playlist that tries to traverse *above* that folder (using relative paths) will look invalid from the perspective of MASS.

**Comment by sebmuc99** [comment]
All music Files are located here (no Folders for Arist or whatever):
/volume1/media01/musicLibrary/musicLibrary/

All Playlists are located here: 
/volume1/media01/musicLibrary/musicLibrary/_PLAYLISTS

thats my mounting: 
/volume1/media01/musicLibrary/musicLibrary:/media

Sample1 is working: located in /media/_PLAYLISTS/Happy Favorits 2.m3u
../FILE003372.mp3
../FILE004255.mp3

Sample2 is working (wired because the path is not correct): located in /media/_PLAYLISTS/Happy Favorits 3.m3u
FILE003372.mp3
FILE004255.mp3

Sample3 is NOT working: located in /media/_PLAYLISTS/Allgemein/Happy Favorits.m3u
../../FILE000996.mp3
../../FILE004803.mp3

**Comment by OzGav** [comment]
Based on the above it seems the title of this report is incorrect. For paths that only go up one level it seems to work but when going up two levels it does not. Is that the case?

**Comment by Rudd-O** [comment]
> Sample3 is NOT working: located in /media/_PLAYLISTS/Allgemein/Happy Favorits.m3u
> ../../FILE000996.mp3
> ../../FILE004803.mp3

Yes, that's not gonna work because it's trying to go above /media.

**Comment by OzGav** [comment]
@Rudd-O i think it goes to media doesn’t it?

**Comment by Rudd-O** [comment]
Oh!  I stand corrected.  Thanks.

**Comment by OzGav** [comment]
@sebmuc99 Please confirm if this is still and issue in beta 14

**Comment by OzGav** [comment]
We will close this soon due to no follow up

**Comment by sebmuc99** [comment]
Sorry needed some time to setup a beta environment. issue still exists with version 2.1.0rc3

**Comment by OzGav** [comment]
This has been fixed in beta 28
